source/features/ci-link.tsx

```diff
@@ -38,5 +70,14 @@ void features.add(__filebasename, {
 		pageDetect.isEmptyRepo
 	],
 	awaitDomReady: false,
-	init
+	init: initRepo
+}, {
+	include: [
+		pageDetect.isPR
+	],
+	additionalListeners: [
+		onConversationHeaderUpdate
+	],
+	awaitDomReady: pageDetect.isPRCommitList(),
```

Comment:
TS help please

Reply:
Can't be done. `pageDetect.isPRCommitList` would only be called on the first page load, but not on Ajax. You'll have to use `false` here and if `domReady` is needed conditionally, add `if (pageDetect.isPRCommitList()) await domLoaded` in the init function